### ISSUE TITLE
Informations over metering operator development status

### ISSUE THREAD

**Issue by JonRiv** [kube-reporting/metering-operator#1460]
Hi there,

we are users of metering operator which we use over at [rating operator](https://github.com/alterway/rating-operator), as a storage back-end. 

We are wondering the reasons why its development is stopped, and if this means that it will be deprecated in the short / middle term.

Thank you for the answer,
Kind regards,
Jon

**Comment by chargio** [comment]
We didn't have enough resources to maintain it so we have decided to stop investing in it. It has already been deprecated and will stop being part of OpenShift (and supported) in a few releases, with only maintenance being done until then.

We will be very happy if you want to fork it and start maintaining it yourselves. We believe it is a fine project that was giving value, but it is just too complex to install and maintain so we needed to take a decision and we decided on another path, where we query prometheus directly and store it, instead of using the operator and thus Presto and Hive.

Kind regards,
Sergio
